### ISSUE TITLE
LESS version of inuit.css v5.0 web-template

### ISSUE THREAD

**Issue by nicoespeon** [inuitless/inuit.css#4]
As you can see, there is a new approach with the framework since `v5.0`. 

Do you think we should to build a LESS version of the web-template? I can help you in that way. The fact is that I reach the `v5.0` in my own repo, but I still not have tagged it because of that particular point.

Otherwise, I'll let you first check about the pull request I send you before. There's no hurry and I don't want you feel upset with all of that amazing work =)

Thanks and regards!

**Comment by peterwilsoncc** [comment]
I haven't had a chance to look over v5 yet.

Users can download the tagged versions, so if LESS supports everything I don't see an issue with continuing.

**Comment by nicoespeon** [comment]
You're right, there is no issue indeed.

In short, Harry changed the way people download the framework for their project in v5, making a distinction between the [web template](https://github.com/csswizardry/inuit.css-web-template) and the [framework core](https://github.com/csswizardry/inuit.css).

The idea is that people don't have to change the inuit.css core anymore. It could be used as a git submodule.

The web template is just a bunch of files containing:
- The core framework as a submodule
- A `_var.scss` file where you actually configure the framework (to override defaults)
- The main `style.scss` which imports the framework, the `_var.scss` and all your personal files you want to use (your `ui/` basically)
- A `go.sh` script to initialize the whole thing if you need
- Some others stuff (an `index.html` and a `watch.sh` so far)

Personally, I don't need a LESS equivalent to figure out that I just need to setup a `var.less` and `style.less` out of the framework core. But I may add some words about that in the README, just in case. Or create the equivalent repo in LESS... What do you think?

**Comment by nicoespeon** [comment]
Well, I just forked Harry's repo and LESSified it to create [the LESS port of inuit.css web template](https://github.com/nicoespeon/inuit.css-web-template).

Then, I updated the core README and get the code up-to-date with `v5.0`. 
I checked and corrected the _units_ thing for the LESS version and it seems to work perfectly now.

I'm about to push on this repo in order to synchronize everything (the web template use this official LESS repo for the submodule, so it needs to be `v5.0-LESS` at least).

Just let me know if I did anything wrong, but it seems to be good so far =)
